### PR TITLE
Content explorer UI fixes

### PR DESCRIPTION
**Pull request by shadyvb**
Fixes https://github.com/humanmade/product-dev/issues/1158

### CODE REVIEW

File: src/data/periods.ts [hunk 1]
```diff
@@ -5,25 +5,29 @@ import { Duration } from '../util';
...
 	label: string,
+	period_label: string,
 	value: Duration,
 	interval: string,
 }[];
 
 export const periods: Periods = [
 	{
 		label: __( '7 Days', 'altis' ),
+		period_label: __( '7-Day', 'altis' ),
```
Comment (robindevitt) [inline src/data/periods.ts:16]:
```suggestion
		period_label: __( '7-Days', 'altis' ),
```

File: src/data/periods.ts [hunk 2]
```diff
@@ -5,25 +5,29 @@ import { Duration } from '../util';
...
 		value: 'P7D',
 		diff: 'P7D',
 		interval: '4h',
 	},
 	{
 		label: __( '30 Days', 'altis' ),
+		period_label: __( '30-Day', 'altis' ),
```
Comment (robindevitt) [inline src/data/periods.ts:23]: Should this not be `30-Days`?
```suggestion
		period_label: __( '30-Days', 'altis' ),
```

File: src/data/periods.ts [hunk 3]
```diff
@@ -5,25 +5,29 @@ import { Duration } from '../util';
...
 		value: 'P30D',
 		diff: 'P30D',
 		interval: '1d',
 	},
 	{
 		label: __( '90 Days', 'altis' ),
+		period_label: __( '90-Day', 'altis' ),
```
Comment (robindevitt) [inline src/data/periods.ts:30]: Should this not be `90-Days`?
```suggestion
		period_label: __( '90-Days', 'altis' ),
```

**Review by robindevitt — COMMENTED** [review]
@shadyvb just a question/s around the `period_label` values.

**Comment by shadyvb** [comment]
Yup, intentional. That's the requirement, check the original issue.

**Review by robindevitt — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**